Applications/DisplayProperties/DisplayProperties.cpp

```diff
@@ -0,0 +1,145 @@
+#include <AK/StringBuilder.h>
+#include <LibCore/CDirIterator.h>
+#include <LibGUI/GApplication.h>
+#include <LibGUI/GAction.h>
+#include <LibGUI/GBoxLayout.h>
+#include <LibGUI/GButton.h>
+#include <LibGUI/GDesktop.h>
+#include <LibGUI/GEventLoop.h>
+#include <LibGUI/GFileSystemModel.h>
+#include <LibGUI/GGroupBox.h>
+#include <LibGUI/GListView.h>
+#include <LibGUI/GScrollBar.h>
+#include <LibGUI/GSplitter.h>
+#include <LibGUI/GTabWidget.h>
+#include <LibGUI/GToolBar.h>
+#include <LibGUI/GWindow.h>
+#include <LibGUI/GWidget.h>
+
+#include <Servers/WindowServer/WSWindowManager.h>
+
+#include "DisplayProperties.h"
+#include "ItemListModel.h"
+
+DisplayPropertiesWidget::DisplayPropertiesWidget()
+    : m_wm_config(CConfigFile::get_for_app("WindowManager"))
+{
+    create_root_widget();
+    create_frame();
+    create_resolution_list();
+    create_wallpaper_list();
+}
+
+void DisplayPropertiesWidget::create_resolution_list()
+{
+    m_resolutions.append({640, 480});
+    m_resolutions.append({800, 600});
+    m_resolutions.append({1024, 768});
+    m_resolutions.append({1280, 1024});
+    m_resolutions.append({1366, 768});
+
+    m_selected_res = m_resolutions.at(0);
```

Comment:
This is problematic. The app starts up with 640x480 selected *on the second tab*, but say you just want to change the wallpaper so you never even switch to the second tab. You pick the wallpaper you like, press _Apply_ and suddenly your whole QEMU window shrinks.

Reply:
> This is problematic. The app starts up with 640x480 selected on the second tab, but say you just want to change the wallpaper so you never even switch to the second tab. 

Whoops, I forgot I had that there. I couldn't really come up with a good method of autodetection, though maybe have `m_resolutions.at(0)` be the currently selected resolution?

Reply:
How about we only *apply* the settings in the visible tab?

Reply:
> How about we only _apply_ the settings in the visible tab?

I like that idea. I've modified `GTabWidget` accordingly to allow for acquiring the currently active tab (as an index).